modules/tinymce/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - New `expand` function added to `tinymce.selection` which expands the selection around the nearest word. #TINY-9001
 - New `expand` function added to `tinymce.dom.RangeUtils` to return a new range expanded around the nearest word. #TINY-9001
+- New `addView` function to `editor.ui.registry` which makes it possible to register custom editor views. #TINY-9210
+- New `ToggleView` command which makes it possible to show/hide registered custom views. #TINY-9210
```

Comment:
```suggestion
- New `ToggleView` command which makes it possible to hide or show registered custom views. #TINY-9210
```

A small style or expression preference here: the _x/y_ construction doesn’t read as cleanly as the more explicit _x or y_.

Also, _x or y_ is generally easier for translators. _x/y_ doesn’t exist as a grammatical norm in some languages, so avoiding it slightly reduces a translator’s ‘gotta re-write that’ work. I know it’s not currently an issue for Tiny docs, but it never hurts to be ready for such an eventuality.

I also reversed the order of the two options in the edit. The norm, in English, is to list *hide* before *show*.

It’s not an overwhelming norm. About 60% of the 3.1 million

```
[hide | show] [conjunction or conjoining glyph] [show | hide]
```

examples in the OED corpora put *hide* before *show*. But it’s norm enough.

Reply:
Oops, thanks for catching this. Yeah we normally don't allow `/` in changelog entries because of the confusion, etc...

Reply:
Old habits been using the / for 10+ years in changelogs so one day when we have a changelog validation in place I might add a rule for my self here since I will likely keep doing this.